src/main/java/net/Winston/vegan_plants/datagen/VeganPlantsItemModelProvider.java

```diff
@@ -0,0 +1,29 @@
+package net.Winston.vegan_plants.datagen;
+
+import net.Winston.vegan_plants.VeganPlants;
+import net.Winston.vegan_plants.item.ItemRegistry;
+import net.minecraft.data.PackOutput;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraftforge.client.model.generators.ItemModelProvider;
+import net.minecraftforge.common.data.ExistingFileHelper;
+
+public class VeganPlantsItemModelProvider extends ItemModelProvider {
+    public VeganPlantsItemModelProvider(PackOutput output, ExistingFileHelper existingFileHelper) {
+        super(output, VeganPlants.MOD_ID, existingFileHelper);
+    }
+
+    @Override
+    protected void registerModels() {
```

Comment:
it looks like the formatter could use some help here. perhaps make local variables for your two `ResourceLocation`s